tim/cli.py

```diff
@@ -334,3 +339,61 @@ def bulk_delete(ctx: click.Context, index: str, source: str, filepath: str) -> N
         results["deleted"],
         results["total"],
     )
+
+
+@main.command()
+@click.option("-i", "--index", help="Name of the index to bulk index records into.")
```

Comment:
Should this refer to `bulk update` rather than just `bulk index`?

Reply:
+1 - and probably needs a slight rewording just to be clear it's indexing + deleting.

Reply:
Just updated!